mteb/cli/build_cli.py

```diff
@@ -361,6 +361,72 @@ def _add_create_meta_parser(subparsers) -> None:
     parser.set_defaults(func=_create_meta)
 
 
+def _add_leaderboard_parser(subparsers) -> None:
+    parser = subparsers.add_parser("leaderboard", help="Launch the MTEB leaderboard")
+
+    parser.add_argument(
+        "--cache-path",
+        type=str,
+        help="Path to the cache folder containing model results",
+        required=False,
+        default=None,
+    )
+    parser.add_argument(
+        "--host",
+        type=str,
+        default="127.0.0.1",
+        help="Host to run the leaderboard server on",
+    )
+    parser.add_argument(
+        "--port",
+        type=int,
+        default=7860,
+        help="Port to run the leaderboard server on",
+    )
+    parser.add_argument(
+        "--share",
+        action="store_true",
+        default=False,
+        help="Create a public URL for the leaderboard",
+    )
+
+    parser.set_defaults(func=_leaderboard)
+
+
+def _leaderboard(args: argparse.Namespace) -> None:
+    """Launch the MTEB leaderboard with specified cache path."""
+    # Import leaderboard module only when needed to avoid requiring leaderboard dependencies
+    # for other CLI commands
+    try:
+        from mteb.leaderboard import get_leaderboard_app
+    except ImportError:
+        raise ImportError(
+            "Leaderboard dependencies are not installed. "
+            "Please install with: pip install mteb[leaderboard]"
+        )
```

Comment:
```suggestion
    except ImportError as e:
        raise ImportError(
            "Seems like some dependencies are not installed. " + 
            "You can likely install these using: `pip install mteb[leaderboard]`. " + 
            f"{e}"
        )
```

Reply:
`likely` comes off as unsure. Would prefer to keep as is.